components/tools/pom.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>edu.jhuapl.dorset.components</groupId>
+    <artifactId>dorset-components</artifactId>
+    <version>0.4.0-SNAPSHOT</version>
+  </parent>
+  <groupId>edu.jhuapl.dorset.components</groupId>
```

Comment:
Just a picky comment: groupId is inherited from the parent so this isn't needed.